ccloud/README.md

```diff
@@ -31,7 +32,18 @@ It steps through the following workflow.
 * Run a Java consumer: showcase a Wildcard ACL
 * Delete the API key, service account, Kafka topics, Kafka cluster, environment, and the log files
 
-# Advanced Demo
+
+# Cloud ETL
+
+The [cloud ETL demo](https://docs.confluent.io/current/tutorials/examples/cloud-etl/docs/index.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.ccloud) showcases a cloud ETL solution leveraging all fully-managed services on Confluent Cloud.
+Using Confluent Cloud CLI, the demo creates a source connector that reads data from an AWS Kinesis stream into Confluent Cloud, then a Confluent Cloud KSQL application processes that data, and then a sink connector writes the output data into cloud storage in the provider of your choice (one of GCP GCS, AWS S3, or Azure Blob).
+
+The documentation for running this demo is at [https://docs.confluent.io/current/tutorials/examples/cloud-etl/docs/index.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.ccloud](https://docs.confluent.io/current/tutorials/examples/cloud-etl/docs/index.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.ccloud)
```

Comment:
IMO the long URLs are unattractive. For comparison see below under hybrid cloud where we have the documentation URL w/ out th UTM codes in the display text.

Reply:
Good catch.  This was a mistake, the URL shown to the user should not include UTM.